database/nakadi/10_data/04_tables/02_subscription.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS zn_data.subscription (
+  s_id varchar(255) NOT NULL PRIMARY KEY CHECK (s_id <> ''),
```

Comment:
In API this is String. And I thought that we will generate it in Nakadi (e.g. UUID)
The max length I just forgot to adjust, will change it to 36.